test/delegatetest.html

```diff
@@ -172,30 +172,20 @@ <h2>Submit Tests</h2>
 	$("#changes thead td").each(function(){
 		var id = "#"+this.id,
 			$cell = $("<td></td>");
-		if ( api == "onX" ) {
+		if ( api === "onX" ) {
 			$(this).find("input, button, select, textarea").each(function(){
 				this["on"+type] = function(e){ e = $.event.fix(e||event); e.data = $cell; blinker.call(this, e); };
 			});
-		} else if ( api == "bind" ) {
-			$(this).find("input, button, select, textarea").bind(type, $cell, blinker);
+		} else if ( api === "bind" ) {
+			$(this).find("input, button, select, textarea").on(type, $cell, blinker);
 		} else {
-			$(id+" input,"+id+" button,"+id+" select,"+id+" textarea").live(type, $cell, blinker);
```

Comment:
This file has been broken for a long time due to the use of `live`… But it looks potentially useful so I fixed it.